cli/src/extension_api.ts

```diff
@@ -1,5 +1,7 @@
 // deno-lint-ignore-file ban-types
 
+/// <reference types="https://raw.githubusercontent.com/denoland/deno/v1.28.3/core/lib.deno_core.d.ts" />
```

Comment:
Wait does this comment actually do something? Are the triple slashes intentional?

Reply:
Yes and yes

<https://deno.land/manual@v1.28.3/advanced/typescript/types#using-a-triple-slash-directive>

Reply:
I should clarify. It is only used during type checking. It will not download that file when run as a phylum extension (and our module loader wouldn't allow it if it tried).

Very weird that TypeScript separates the idea of type checking and compiling... Plenty of code will happily compile even if it doesn't type check correctly.